lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -699,27 +700,31 @@ SwiftLanguageRuntimeImpl::emplaceClangTypeInfo(
     CompilerType clang_type, llvm::Optional<uint64_t> byte_size,
     llvm::Optional<size_t> bit_align,
     llvm::ArrayRef<swift::reflection::FieldInfo> fields) {
-  std::lock_guard<std::recursive_mutex> locker(m_clang_type_info_mutex);
+  const std::lock_guard<std::recursive_mutex> locker(m_clang_type_info_mutex);
   if (!byte_size || !bit_align) {
     m_clang_type_info.insert({clang_type.GetOpaqueQualType(), llvm::None});
     return nullptr;
   }
   assert(*bit_align % 8 == 0 && "Bit alignment no a multiple of 8!");
   auto byte_align = *bit_align / 8;
   // The stride is the size rounded up to alignment.
-  size_t byte_stride = llvm::alignTo(*byte_size, byte_align);
+  const size_t byte_stride = llvm::alignTo(*byte_size, byte_align);
+  auto extra_inhabitants = 
+      clang_type.IsPointerOrReferenceType()
```

Comment:
Can imported types be reference types? Wondering if this can be `IsPointerType()`?

Reply:
For C and ObjC no, but for C++ we should find out whether that's a possibility.